### PR TITLE
add index.html style.css

### PR DESCRIPTION
**Pull request by OlgaBarnikova8**
### HTML і CSS практика: Hooli-style Popup

[Demo](https://olgabarnikova8.github.io/hooli-style-popup/)
[Code base](https://github.com/OlgaBarnikova8/kottans-frontend/tree/main/Popup)

The code is submitted in a dedicated feature branch.

Only code files are submitted.

Please, review.

### CODE REVIEW

**Comment by OlgaBarnikova8** [comment]
Пожалуйста проверьте.

**Comment by OlgaBarnikova8** [comment]
Please, review.

File: submissions/OlgaBarnikova/Popup/index.html [hunk 1]
```diff
@@ -0,0 +1,157 @@
...
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>HTML-CSS-Popup</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+    <nav class="navigation">
```
Comment (A-Ostrovnyy) [inline submissions/OlgaBarnikova/Popup/index.html:11]: Please fix mobile version
image

File: submissions/OlgaBarnikova/Popup/index.html [hunk 2]
```diff
@@ -0,0 +1,157 @@
...
+                              <img src="popup-icons/videos.png" alt="" 
+                              class="popup_img" />
+                              <div>Videos</div>
+                            </a>                            
+                         </li>   
+                    </ul>
+                    <ul class="popup_more">
+                        <li>
+                        <input id="popup-more" type="checkbox" class="popup_more_input">
+                        <label for="popup-more" class="popup_more_icon">More</label>
```
Comment (A-Ostrovnyy) [inline submissions/OlgaBarnikova/Popup/index.html:115]: Add styles on focus state. Also, hide this button when the second part of the popup is open.

File: submissions/OlgaBarnikova/Popup/index.html [hunk 3]
```diff
@@ -0,0 +1,157 @@
...
+                    class="popup_img" /></a>
+            </li>
+            <li class="navigation_item">
+                <a href="#"><img src="popup-icons/bell.png" alt="" 
+                    class="popup_img" /></a>
+            </li>
+        </ul>
+    </nav>
+</body>
+</html>
```
Comment (A-Ostrovnyy) [inline submissions/OlgaBarnikova/Popup/index.html:157]: Add an empty line at the end of each file. [Reason](https://stackoverflow.com/questions/729692/why-should-text-files-end-with-a-newline?noredirect=1&lq=1)

File: submissions/OlgaBarnikova/Popup/style.css [hunk 4]
```diff
@@ -0,0 +1,115 @@
...
+    box-sizing: border-box;     
+    list-style: none; 
+}
+
+a, a:active {
+    text-decoration: none;
+    color: rgb(94, 94, 226);
+}
+
+:focus {
```
Comment (A-Ostrovnyy) [inline submissions/OlgaBarnikova/Popup/style.css:11]: This is an invalid CSS selector: you can't set pseudoclass to nothing, you should provide an element:
```CSS
.block:focus {}

.a:focus {}

*:focus {}
```

**Review by A-Ostrovnyy — CHANGES_REQUESTED** [review]
Good work!

**Comment by OlgaBarnikova8** [comment]
HTML і CSS практика: Hooli-style Popup
[Demo] https://olgabarnikova8.github.io/hooli-style-popup/
[Code base] https://github.com/OlgaBarnikova8/kottans-frontend/tree/main/Popup

Thanks for your work!
Please, review.

File: submissions/OlgaBarnikova/Popup/index.html [hunk 5]
```diff
@@ -0,0 +1,119 @@
...
+    <body>
+        <div class="wrapper">
+            <header class="header">  
+                <div class="block_menu">
+                    <nav class="navigation">
+                        <a href="#" class="popup_icons">Proekt</a>
+                        <a href="#" class="popup_icons">About us</a>
+                        <label for="popup">
+                            <img src="popup-icons/popup-button.png" alt="" class="popup_block_img" />                            
+                        </label>
```
Comment (A-Ostrovnyy) [inline submissions/OlgaBarnikova/Popup/index.html:18]: Users must have the ability to open a popup with a keyboard. [Watch video](https://share.vidyard.com/watch/ZDCPyrd5EAnKvP895smxQc?)

File: submissions/OlgaBarnikova/Popup/index.html [hunk 6]
```diff
@@ -0,0 +1,119 @@
...
+                                <li>
+                                    <a href="#" class="popup_block_img"><img src="popup-icons/videos.png" alt="" 
+                                        class="popup_img" /></a>  
+                                    <div>Videos</div>   
+                                </li>
+                            </ul>
+                            <label for="popup-more">
+                                <div class="popup_more_name">more...</div>                      
+                            </label>
+                            <input type="checkbox" id="popup-more" class="popup_input_more" />
```
Comment (A-Ostrovnyy) [inline submissions/OlgaBarnikova/Popup/index.html:86]: Add styles on focus state, also, hide this button when the second part of the popup is open. [Watch video](https://share.vidyard.com/watch/ZDCPyrd5EAnKvP895smxQc?)

**Review by A-Ostrovnyy — CHANGES_REQUESTED** [review]
Good progress!

**Comment by OlgaBarnikova8** [comment]
HTML і CSS практика: Hooli-style Popup
[Demo](https://olgabarnikova8.github.io/hooli-style-popup/) |
[Code base](https://github.com/OlgaBarnikova8/kottans-frontend/tree/main/Popup)

Thanks for your work!
Please, review.

**Review by A-Ostrovnyy — APPROVED** [review]
Well done!

**Pull request merged**